test/parallel/test-fs-read-type.js

```diff
@@ -118,7 +118,10 @@ fs.read(fd,
         2n ** 53n - 1n,
         common.mustCall((err) => {
           if (err) {
-            assert.strictEqual(err.code, 'EFBIG');
+            if (common.isIBMi)
+              assert.strictEqual(err.errno, -127);
```

Comment:
What error does -127 map to on IBM i? Should we try to map it in libuv?

Reply:
It is `#define EOVERFLOW       127     /* value too large to be stored in data type */`

Reply:
> What error does 127 map to on IBM i? Should we try to map it in libuv?

@cjihrig PR to add a mapping for `EOVERFLOW`: https://github.com/libuv/libuv/pull/3145